pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdBrokerStats.java

```diff
@@ -139,8 +144,4 @@ public CmdBrokerStats(PulsarAdmin admin) {
         jcommander.addCommand("load-report", new CmdLoadReport());
     }
 
-    public void addCommands(String commandName, CliCommand cliCommand) {
```

Comment:
There is no one using this?

Reply:
yeah, idea shows this method is never used.